src/main/target/CC3D/target.c

```diff
@@ -26,17 +26,19 @@
 
 const timerHardware_t timerHardware[USABLE_TIMER_CHANNEL_COUNT] = {
 
-    DEF_TIM(TIM4, CH1, PB6, TIM_USE_PWM,               0),   // S1_IN/
+//    DEF_TIM(TIM4, CH1, PB6, TIM_USE_PWM,               0),   // S1_IN/  <--Moved to allow Parallel PWM with proper ESC
     DEF_TIM(TIM3, CH2, PB5, TIM_USE_PWM,               0),   // S2_IN - SoftSerial TX - GPIO_PartialRemap_TIM3 / Sonar trigger
     DEF_TIM(TIM3, CH3, PB0, TIM_USE_PWM,               0),   // S3_IN - SoftSerial RX / Sonar echo / RSSI ADC
     DEF_TIM(TIM3, CH4, PB1, TIM_USE_PWM,               0),   // S4_IN - Current
     DEF_TIM(TIM2, CH1, PA0, TIM_USE_PWM,               0),   // S5_IN - Vbattery
-    DEF_TIM(TIM2, CH2, PA1, TIM_USE_PWM | TIM_USE_PPM, 0),   // S6_IN - PPM IN
```

Comment:
Why did you remove `TIM_USE_PPM` here? I think PPM should still work on this pin.

Reply:
You are right.  When I was digging in to find out why it wasn't working I wasn't sure if the 'TIM_USE_PPM' was part of the problem.  It really should have that back in the code.

I will edit and redo the pull request.

Reply:
All good now, no need to do anything more. Pushing a new changeset into a branch with an open pull request will automatically update the pull request to use the latest version in the branch.